server/middleware/setUpSentry.ts

```diff
@@ -28,19 +22,13 @@ function setUpSentryRequestHandler(app: express.Express): void {
       },
       dsn: config.sentry.dsn,
       environment: config.environment,
-      integrations: [
```

Comment:
As per https://docs.sentry.io/platforms/javascript/guides/express/migration/v7-to-v8/#updated-sdk-initialization, these integrations were changed to be supported out of the box in v8 (which we've skipped over).